### DISCUSSION TITLE
How to change information at bottom of meeting listing

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#1456] [Q&A]
Hi,

I need to change the information at the bottom of a meeting listing, but I am not sure where to do this in the plugin. Is it a setting somewhere? 

More specifically - I need to change the link in the hyperlink for the online Group Records form
<img width="1040" alt="Screenshot 2024-06-23 at 9 19 07 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/59859095/367aaf38-5bc9-4850-bbae-760999c8e969">

**Comment by joshreisner** [comment]
this doesn't look like it's our plugin. maybe it's in a widget? try going to Appearance > Widgets and looking for Meeting Bottom

**Comment by OSAwebsite** [comment]
Thank you!
